synthesizer/src/vm/execute.rs

```diff
@@ -154,7 +171,7 @@ impl<N: Network, C: ConsensusStorage<N>> VM<N, C> {
                 lap!(timer, "Compute the proof");
 
                 // Return the execution.
-                Ok(cast_ref!(execution as Execution<N>).clone())
+                Ok((cast_ref!(execution as Execution<N>).clone(), cast_ref!(response as Response<N>).clone()))
```

Comment:
I haven't gone through these macros yet but do you know why these clones are necessary? 🕵️‍♂️

Reply:
Thanks for the context! Perhaps something for later, in that case 👍